src/main/java/org/alfresco/repo/domain/schema/SchemaBootstrap.java

```diff
@@ -924,21 +961,40 @@ private boolean updateSchema(Connection connection) throws Exception
                 appliedPatch.setReport("Placeholder for Activiti bootstrap at schema " + installedSchemaNumber);
                 appliedPatchDAO.createAppliedPatch(appliedPatch);
             }
+            if (logger.isInfoEnabled())
+            {
+                logger.info("Creating and patching activiti tables executed in: " + (System.currentTimeMillis() - start) + " ms");

```

Comment:
Please use the past tense for messages after the fact:
"Creating and patching Activiti tables took 121345ms"